File: src/main/java/com/swp391/koibe/repositories/AuctionParticipantRepository.java
package com.swp391.koibe.repositories;

import com.swp391.koibe.models.AuctionParticipant;
import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface AuctionParticipantRepository extends JpaRepository<AuctionParticipant, Long> {
    AuctionParticipant getAuctionParticipantByAuctionIdAndUserId(Long auctionId, Long userId);
    List<AuctionParticipant> getAuctionParticipantsByAuctionId(Long auctionId);
}

File: src/main/java/com/swp391/koibe/repositories/AuctionParticipantRepository.kt
package com.swp391.koibe.repositories

import com.swp391.koibe.models.AuctionParticipant
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface AuctionParticipantRepository : JpaRepository<AuctionParticipant, Long> {
    fun getAuctionParticipantByAuctionIdAndUserId(auctionId: Long, userId: Long): AuctionParticipant?
    fun getAuctionParticipantsByAuctionId(auctionId: Long): List<AuctionParticipant>
}
